std_msgs/Header header

# The name of this finger
uint8 type

# Reports whether the finger is more or less straight
uint8 is_extended

# List of bones associated with this finger
Bone[] bone_list